examples/disable-persistence-queue-logs/disable-persistence-queue-logs-values.yaml

```diff
@@ -0,0 +1,31 @@
+clusterName: CHANGEME
+splunkPlatform:
+  endpoint: CHANGEME
+  token: CHANGEME
+  index: CHANGEME
+  metricsIndex: CHANGEME
+  metricsEnabled: true
+  tracesEnabled: true
+  logsEnabled: true
+  sendingQueue:
+    persistentQueue:
+      enabled: true
+      storagePath: "/var/addon/splunk/persist"
+
+logsEngine: otel
+
+agent:
+  securityContext:
+    runAsUser: 999
+    runAsGroup: 999
+  config:
+    exporters:
+       splunk_hec/platform_logs:
+         sending_queue:
+           storage: null
```

Comment:
Let's not create an example per each possible configuration. It's too much to maintain and not that helpful for users. Can we have only one example where we enable the persistent queue for logs and traces and disable it for metrics (seems like the most reasonable scenario)?